common/static/common/js/spec/discussion/view/discussion_topic_menu_view_spec.js

```diff
@@ -16,67 +16,67 @@
                                 },
                                 'Very long category name': {
-                                    'sort_key': null,
-                                    'is_cohorted': false,
-                                    'id': 'c49f0dfb8fc94c9c8d9999cc95190c59'
+                                    sort_key: null,
+                                    is_divided: false,
+                                    id: 'c49f0dfb8fc94c9c8d9999cc95190c59'
                                 },
                                 'Very very very very long category name': {
-                                    'sort_key': null,
-                                    'is_cohorted': false,
-                                    'id': 'c49f0dfb8fc94c9c8d9999cc95190e32'
+                                    sort_key: null,
+                                    is_divided: false,
+                                    id: 'c49f0dfb8fc94c9c8d9999cc95190e32'
                                 },
                                 'Name with <em>HTML</em>': {
-                                    'sort_key': null,
-                                    'is_cohorted': false,
-                                    'id': 'c49f0dfb8fc94c9c8d9999cc95190363'
+                                    sort_key: null,
+                                    is_divided: false,
+                                    id: 'c49f0dfb8fc94c9c8d9999cc95190363'
                                 }
 
                             }
                         },
                         'Example Inline Discussion': {
-                            'subcategories': {},
-                            'children': [
+                            subcategories: {},
+                            children: [
                                 ['What Are Your Goals for Creating a MOOC?', 'entry']
                             ],
-                            'entries': {
+                            entries: {
                                 'What Are Your Goals for Creating a MOOC?': {
-                                    'sort_key': null,
-                                    'is_cohorted': true,
-                                    'id': 'cba3e4cd91d0466b9ac50926e495b931'
+                                    sort_key: null,
+                                    is_divided: true,
+                                    id: 'cba3e4cd91d0466b9ac50926e495b931'
                                 }
                             }
                         }
                     },
-                    'children': [ // eslint-disable-line quote-props
+                    children: [
                         ['Basic Question Types', 'subcategory'],
                         ['Example Inline Discussion', 'subcategory']
                     ],
-                    'entries': {}
+                    entries: {}
                 },
-                'is_cohorted': true
+                is_cohorted: true
```

Comment:
Note to reviewers: this general setting about whether or not the course is cohorted has not yet changed. However, the "is_cohorted" value for individual threads did change, as you have seen above.